### DISCUSSION TITLE
Relationships

### DISCUSSION THREAD

**Discussion by digitalorbitsa** [Budibase/budibase#3566] [Help] [answered]
I have a Table with Patient Details, now i want other tables to be related to it such as Person Responsible, Medical Aids etc so now when i create relationships to the Patient Details table which has a primary key file number it doesn't create the column instead it brings me here.
![Screenshot 2021-11-27 at 17 06 42](https://user-images.githubusercontent.com/94488681/143686726-472a5052-be46-4150-a6be-3527993681ae.png)
..

**Comment by digitalorbitsa** [comment]
![Screenshot 2021-11-27 at 17 08 55](https://user-images.githubusercontent.com/94488681/143686791-b4e04595-34b4-4987-80ee-eb5819f41cc9.png)

**Comment by digitalorbitsa** [comment]
How do you related one table to multiple tables?

**Comment by mjashanks** [comment, marked as answer]
It should be possible to create multiple relationships to the same table.

Regarding your `Column name in other table` - are you trying to set that to `File number` each time? That could be the issue - conflicting column names.

In your screenshot, I think you should be calling it `Nearest Family / Friends` - i.e. this is how the relationship will appear in the `File` table. 

We should report this error better of course!

**Reply by digitalorbitsa** [reply]
I understand you now, couldn't go around it at first especially on how to explain the issue. Thanks a lot.
